.github/workflows/ci.yml

```
@@ -12,6 +12,13 @@ env:
   NODE_OPTIONS: --max-old-space-size=6144
 
 jobs:
+  commitlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: wagoid/commitlint-github-action@v5
```

suggestion: link to our commit message guidelines
```suggestion
      - uses: wagoid/commitlint-github-action@v5
        with:
          helpURL: https://github.com/analogjs/analog/blob/main/CONTRIBUTING.md#-commit-message-guidelines
```

---

@LayZeeDK Done, thanks!